client/src/pages/admin/tasks.tsx

```diff
@@ -63,6 +64,21 @@ function Page(props: Props) {
 
   const handleModalSubmit = useCallback(
     async (values: any) => {
+      const checkCriteria = () => {
+        return dataCriteria.every(item => {
+          if (item.type !== TaskType.Title) {
+            return item.max !== 0;
+          }
+          return;
+        });
+      };
```

Comment:
1. `return` on line 72 is unnecessary, we could safely remove it
2. It seems that we can add penalty with positive score, please consider to add one more check for this case

Reply:
For the first note - actually I missed `true`  :(
For the second - for these cases to avoid positive penalties we use only the absolute value of a number for following calculating the final score